<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TruncarArquivosTexto</class>
 <widget class="QWidget" name="TruncarArquivosTexto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Linhas:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="labelTotalDeLinhas">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>[...]</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="plainTextEdit"/>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="botaoProcessar">
       <property name="text">
        <string>Processar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="botaoSalvar">
       <property name="text">
        <string>Salvar</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>398</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Local do arquivo</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="campoLocalDoArquivo">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>campoLocalDoArquivo</tabstop>
  <tabstop>botaoProcessar</tabstop>
  <tabstop>botaoSalvar</tabstop>
  <tabstop>plainTextEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
